support-e2e/tests/utils/users.ts

```diff
@@ -1,16 +1,27 @@
-import { v4 as uuidv4 } from 'uuid';
+import { v4 as uuidv4 } from "uuid";
 
-const characters ='ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+const characters =
+  "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
 
-function generateString(length:number) {
-    let result = '';
-    const charactersLength = characters.length;
-    for ( let i = 0; i < length; i++ ) {
-        result += characters.charAt(Math.floor(Math.random() * charactersLength));
-    }
-    return result;
+function generateString(length: number) {
+  let result = "";
+  const charactersLength = characters.length;
+  for (let i = 0; i < length; i++) {
+    result += characters.charAt(Math.floor(Math.random() * charactersLength));
+  }
+  return result;
 }
 
 export const firstName = () => `${generateString(5)}TestF`;
 export const lastName = () => `${generateString(5)}TestL`;
```

Comment:
unrelated to this PR, but it's sometimes nicer to have the fixed part at the beginning to aid ordering

Reply:
Great suggestion - [what do you think of this](https://github.com/guardian/support-frontend/pull/5782)?

Reply:
This change has meant we're generating email addresses that are greater than 80 characters long for the GW Gift tests as they also prepend 'giftee' to the email address generated by this function. 80 characters is the maximum email address length accepted by SalesForce, so the e2e tests are failing the GW Gift tests as the `CreateSalesforceContact` step in the `Support-Workers-Prod` step functions is now failing.